src/test/java/tests/FilterTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import indices.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.ApplePage;
+import pages.DominoPage;
+import pages.MainPage;
+import pages.SmartphonePage;
+import testData.Catalogue;
+
+
+public class FilterTest extends BaseTest {
+
+    @Test(description = "Positive test to choose Apple manufacturer in the filter")
+    public void chooseManufacturerTest() {
+        ApplePage applePage = new MainPage()
+                .openPage()
+                .clickOnCatalogue()
+                .getIconOption(Icons.ELECTRONICS)
```

Comment:
why do you need to get icon option here? it's not really understandable

Reply:
renamed